osu.Game.Rulesets.Osu/Objects/Drawables/Connections/FollowPointRenderer.cs

```diff
@@ -74,6 +72,8 @@ private void addConnection(FollowPointConnection connection)
                 FollowPointConnection previousConnection = connections[index - 1];
                 previousConnection.End = connection.Start;
             }
+
+            AddInternal(connection);
```

Comment:
I presume this was moved to the end of the method to ensure connection ends were initialised before proceeding, right?

Reply:
Generally with `LifetimeManagementContainer` you want to set the lifetime *before* adding to avoid an extra update. I checked this and it doesn't seem to matter where this specific call is (since it's all loaded in BDL anyway) but it feels like the more correct place to put it so I left it at the end.